Cubical/Foundations/GroupoidLaws.agda

```diff
@@ -116,9 +116,15 @@ preassoc {x = x} p q r j i = preassoc-filler p q r i1 j i
 
 -- deducing associativity for compPath
 
+assoc' : (p : x ≡ y) (q : y ≡ z) (r : z ≡ w) →
```

Comment:
Can you add a comment explaining that there are 2 proofs of this and which one should be used?

Reply:
the old proofs looks like this:
[assoc] (https://raw.githack.com/marcinjangrzybowski/cubeViz/master/main.html#assoc,assoc)
the new (by ecavallo)
[assocAlt](https://raw.githack.com/marcinjangrzybowski/cubeViz/master/main.html#assocAlt,assoc')

I left the old proof, but it is not clear to me when it would be useful, I can remove it completely, i checked and whole library works without it

Reply:
The pentagonIdentity is mechanical to prove after looking at diagrams at the end of 
[`assocAlt`](https://raw.githack.com/marcinjangrzybowski/cubeViz/master/main.html#assocAlt)
at the bottom of the page you can see the opposite faces of big cube,
ich lemma in the proof represents part where those faces are different.

Reply:
Wow, you just completely blew my mind with these diagrams! How do you draw them?

Reply:
> the old proofs looks like this:
> [assoc] (https://raw.githack.com/marcinjangrzybowski/cubeViz/master/main.html#assoc,assoc)
> the new (by ecavallo)
> https://raw.githack.com/marcinjangrzybowski/cubeViz/master/main.html#assocAlt,assoc'
> 
> I left the old proof, but it is not clear to me when it would be useful, I can remove it completely, i checked and whole library works without it

The new proofs look a lot better indeed, so please just remove the old ones.

Reply:
thanks!
the code for diagrams is here:
https://github.com/marcinjangrzybowski/cubeViz
they are drawn directly from Agda internal representation, 
everything is written in elm for now (i reimplemented small part of agda typechecker),  but  I plan to make it finaly work as emacs plugin,
It is still ongoing work, so i will let you now when it will be finished, 
my goal is to turn it into editor so it will be possible to prove things by assembling diagrams.

Reply:
That's really cool! It would be great with some documentation on how to read the diagrams. I can kind of see it for some of the simpler examples, but for the more complex ones I'm quite lost